Project.toml

```
@@ -36,17 +35,16 @@ DocStringExtensions = "0.8, 0.9"
 LogDensityProblems = "2"
 LogDensityProblemsAD = "1"
 MCMCChains = "5, 6"
-OrdinaryDiffEq = "6"
+OrdinaryDiffEq = "6.92"
```

This seems unnecessary?

```suggestion
OrdinaryDiffEq = "6"
```